src/crypto-browser.js

```diff
@@ -6,14 +8,13 @@ const nodeify = require('nodeify')
 const webCrypto = getWebCrypto()
 
 function getWebCrypto () {
```

Comment:
Any reason why `getWebCrypto` is defined twice in separate files? Should it be moved to some utils lib/file?